src/mudlet.cpp

```diff
@@ -1257,7 +1257,19 @@ bool mudlet::setLabelClickCallback( Host * pHost, const QString & name, const QS
     QMap<QString, TLabel *> & labelMap = mHostLabelMap[pHost];
     if( labelMap.contains( name ) )
     {
-        labelMap[name]->setScript( pHost, func, pA );
+        labelMap[name]->setClick( pHost, func, pA );
```

Comment:
Would it be reasonable to actually pass the return value from the `TLabel::setClick(...)` (and the other setRelease() and probably the enter and leave things as well) to the user?  It indicates (with a true value) that the label was found for the Lua "script" to be allocated to the event IIRC and it would make sense and be consistent to do so, IMHO.